Resources/Prototypes/Entities/Objects/Misc/bedsheets.yml

```diff
@@ -28,23 +28,18 @@
     state: sheetblack
   - type: Clothing
     size: 10
-    QuickEquip: true
-    Slots:
-    - neck
     sprite: Clothing/Neck/Bedsheets/black.rsi
 
 - type: entity
   id: BedsheetBlue
   parent: BedsheetBase
   name: blue bedsheet
+  description:
```

Comment:
Why would you add a description with no entry?

Reply:
Typo